sled-agent/src/services.rs

```diff
@@ -3929,42 +3994,42 @@ impl ServiceManager {
                     );
                 }
 
-                if let Some(info) = self.inner.sled_info.get() {
-                    zone.add_default_route(info.underlay_address).map_err(
-                        |err| Error::ZoneCommand {
-                            intent: "Adding Route".to_string(),
-                            err,
-                        },
-                    )?;
-                }
-
                 for service in &request.services {
                     let smfh = SmfHelper::new(&zone, service);
 
                     match service {
                         SwitchService::ManagementGatewayService => {
+                            info!(self.inner.log, "configuring MGS service");
                             // Remove any existing `config/address` values
                             // without deleting the property itself.
-                            smfh.delpropvalue("config/address", "*")?;
+                            smfh.delpropvalue_default_instance(
```

Comment:
Just adding a note here about why I'm deleting and adding property values on the default instance instead. Previously, all services added property values to the service FMRI. Now with the new PropertyBuilder flow, we are adding them to the default service instance FMRI. This means that if I didn't change these values to be set directly on the default instance itself, we had a mess of some values set on the instance and some values set on the service directly